pcdsdevices/mpod_apalis.py

```diff
@@ -246,6 +297,54 @@ def set_current_ramp_speed(self, ramp_speed):
         """
         self.current_ramp_speed.put(ramp_speed)
 
+    @model.sub_value
+    def _stash_model(self, value, **kwargs) -> None:
+        self._model_str = self.model.get()
```

Comment:
A bit of ignorance, are these run once on connection?  If not then I probably need to initialize these values to something more real (but potentially incorrect)